### DISCUSSION TITLE
The Fuk Latency

### DISCUSSION THREAD

**Discussion by AtharvaxL** [marioparaschiv/nitro-sniper#234] [Q&A] [answered]
when it was hosted on heroku it would give delay like around 70-90ms but through replit free it gives like 500-1000ms per snipe and i also tested on replit premium (hecker plan) and still i got around 200ms. any solutions for that? thanks!

**Comment by MrD4rio** [comment, marked as answer]
Unless you can change location of your replit, I guess there's nothing to do about that. You might want to consider spending 5-10 bucks for a VPS located in the USA.

**Reply by x32u** [reply]
will the 5$ droplet from digitalocean enough ?

**Reply by MrD4rio** [reply]
> will the 5$ droplet from digitalocean enough ?

Not sure if you got already your answer, but I'd say it depends on what specs you get. If you run not many alts I'd say 2gb RAM and 2 Cores would be good enough, but still I'm not sure if you can cover that with $5 credit as I don't know how cheap DigitalOcean is.
You might want to check zap-hosting, they offer a 50% off discount for the first time customers, you can get a very good machine at a cheap price
